src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Compile.xaml

```diff
@@ -38,6 +38,12 @@
       DisplayName="Custom Tool Namespace"
       Description="The namespace into which the output of the custom tool is placed." />
 
+    <StringProperty Name="FullPath" Visible="false">
```

Comment:
What does this represent? Is this relative to the project path? If so I should use this for language service.